File: DbCard/DbCard/Infrastructure/DTO/FilialDTO/FilialForAdd.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace DbCard.Infrastructure.DTO.FilialDTO
{
    public class FilialForAdd
    {
        [Required]
        public string City { get; set; }
        [Required]
        public string Street { get; set; }
        [Required]
        public string HouseNumber { get; set; }
        [Required]
        public string PhoneNumber { get; set; }
        public long PartnerId { get; set; }
    }
}

File: DbCard/Angular/src/app/_models/Filial/FilialForAdd.ts
import {Schedule} from 'src/app/_models/Filial/Schedule';

export interface FilialForAdd
{
    Region: string;
    City: string;
    Street: string;
    HouseNumber: string;
    PhoneNumber: string;
    Schedule: Schedule;
    IsMainOffice: boolean;
    PartnerId: number;
}
